public/js/loginValidation.js

```diff
@@ -0,0 +1,50 @@
+let isValidEmail = (elementID) => {
+    let inputVal = document.getElementById(elementID)
+    let userEmailWarning = document.getElementById('userEmailWarning')
+    let email = inputVal.value.trim();
+    let regEx = new RegExp("^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$")
+    if (regEx.test(email)) {
```

Comment:
this could be shortened to `return regEx.text(email)`

Reply:
done.